spacy/cli/project/assets.py

```diff
@@ -12,6 +12,9 @@
 from .._util import get_checksum, download_file, git_checkout, get_git_version
 from .._util import SimpleFrozenDict, parse_config_overrides
 
+# Whether assets are extra if `extra` is not set.
+OPTIONAL_DEFAULT = False
```

Comment:
Maybe for consistency let's call this `EXTRA_DEFAULT`?

Reply:
Done. Sorry for missing the other `optional`s!